test/Azure.Functions.Cli.Tests/PublishActionTests.cs

```diff
@@ -142,6 +142,19 @@ public async Task NetFrameworkVersion_Invalid(string specifiedVersion)
             Assert.StartsWith($"The dotnet-version value of '{specifiedVersion}' is invalid.", exception.Message);
         }
 
+        [Theory]
+        [InlineData("dotnet-isolated", WorkerRuntime.dotnetIsolated)]
+        [InlineData("c#-isolated", WorkerRuntime.dotnetIsolated)]
+        [InlineData("csharp", WorkerRuntime.dotnet)]
+        [InlineData("typescript", WorkerRuntime.node)]
```

Comment:
should we have some test cases where the inline data is none of the valid values?

Reply:
@aishwaryabh , I have added a new test case to handle invalid values.